src/js/device/DeviceCommands.js

```diff
@@ -278,18 +271,15 @@ export default class DeviceCommands {
         };
     }
 
-    async ethereumGetAddress(
-        { address_n, show_display }: PROTO.EthereumGetAddress,
-        network?: EthereumNetworkInfo,
-    ) {
+    async ethereumGetAddress({ address_n, show_display }: PROTO.EthereumGetAddress) {
```

Comment:
can be removed

Reply:
ah, it is probably useful to keep it here as it does some additional work (path serialization).